### PR TITLE
Fsi updates 111620a

### PR DESCRIPTION
**Pull request by jbranson000**
https://github.com/adobe/xdm/issues/1017

### CODE REVIEW

File: components/mixins/experience-event/experienceevent-bill-pay-steps.schema.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Bill Pay Steps",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:industry": ["financial_services"],
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Bill pay steps such as start, save, failure, and complete.",
+  "properties": {
+    "xdm:financialServices": {
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-bill-pay-steps.schema.json:18]: Should this be "xdm:personalFinances" ?

File: components/datatypes/transaction.schema.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  "meta:extensible": true,
+  "meta:industry": ["financial_services", "travel_and_hospitality", "telecom", "health_and_life_sciences"],
+  "description": "Transactions capture when non-order transactions occur, such as balance transfers or applied payments.",
+  "properties": {
+    "xdm:transactionAmount": {
+      "title": "Transaction Amount",
+      "$ref": "https://ns.adobe.com/xdm/datatypes/currency",
+      "description": "Captures the amount of the transaction."
+    },
+    "xdm:transactionId": {
```
Comment (mjasrotia) [inline components/datatypes/transaction.schema.json:21]: id should be "ID" - (transactionID)

File: components/datatypes/financial-account.schema.json [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    },
+    "xdm:financialAccountType": {
+      "title": "Financial Account Type",
+      "enum": ["checking", "savings", "retirement", "investment"],
+      "description": "Describes the type of finanical account, such as checking, savings, or retirement."
+    },
+    "xdm:openedDate": {
+      "title": "Financial Account Open Date",
+      "type": "string",
+      "format": "date",
```
Comment (mjasrotia) [inline components/datatypes/financial-account.schema.json:44]: "format": "date-time"

File: components/datatypes/financial-account.schema.json [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+      "description": "Current account balance for a single user account."
+    },
+    "xdm:financialAccountName": {
+      "title": "Financial Account Name",
+      "type": "string",
+      "description": "Name assigned to the financial account."
+    },
+    "xdm:financialAccountOwner": {
+      "title": "Financial Account Owners",
+      "type": "array: mixin",
```
Comment (mjasrotia) [inline components/datatypes/financial-account.schema.json:33]: This is not how we define array object. I think this is your VBA issue. Please fix this

File: components/mixins/experience-event/experienceevent-bill-pay-steps.schema.json (hunk 1 above)
Comment (jbranson000) [reply components/mixins/experience-event/experienceevent-bill-pay-steps.schema.json]: Yes.  I've correct that.

File: components/datatypes/transaction.schema.json (hunk 2 above)
Comment (jbranson000) [reply components/datatypes/transaction.schema.json]: Fixed.

File: components/datatypes/financial-account.schema.json [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+      "description": "Array of the owner(s) of the selected financial account. Extension of the personal profile details data type."
+    },
+    "xdm:financialAccountType": {
```
Comment (mjasrotia) [inline components/datatypes/financial-account.schema.json:36]: "type" missing

**Pull request closed without merging**